status_handler.go

```diff
@@ -40,14 +41,42 @@ type ErrorStatusHandler struct {
 	Component
 }
 
-// Handle generic error reponses.
+// Handle generic error responses.
 func (*ErrorStatusHandler) Handle(response *Response, _ *Request) {
 	message := map[string]string{
 		"error": http.StatusText(response.GetStatus()),
 	}
 	response.JSON(response.GetStatus(), message)
 }
 
+// RequestErrorStatusHandler a generic (error) status handler for requests.
+type RequestErrorStatusHandler struct {
+	Component
+}
+
+// Handle generic request (error) responses.
+func (h *RequestErrorStatusHandler) Handle(response *Response, request *Request) {
+	var errorMessage string
```

Comment:
You should initialize it to "unexpected error"

Otherwise you might have an empty message if you don't enter the if ok

Reply:
I don't really want to give yet another "unexpected error". I'm hoping to cover all scenarios now :)